src/Utility/DspParams.cpp

```diff
@@ -229,7 +222,25 @@ void DspParams::initIntParams()
 	IntParams_.createParam("DE/SOLVER", OsiCpx);
 	IntParams_.createParam("DW/MASTER/SOLVER", OsiCpx);
 	IntParams_.createParam("DW/SUB/SOLVER", OsiCpx);
+#else
+	IntParams_.createParam("DW/MASTER/SOLVER", OsiClp);
```

Comment:
Why is this set to `OsiClp`? The master problem is QP that can be solved by `OsiGrb`.